README.md

```diff
@@ -278,6 +278,50 @@ API. The commented interface is available in
 and
 [http](src/python/library/tritonclient/http/__init__.py).
 
+### GRPC Options
+
+#### SSL/TLS
+
+The client library allows a secured channel using gRPC protocol.
```

Comment:
I know I added this,.. But can you fix it to read "The client library allows communication across a secured channel using grpc protocol" ?